app/src/main/res/layout-sw600dp-land/onboarding_profile_type_fragment.xml

```diff
@@ -0,0 +1,151 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto">
+
+  <androidx.constraintlayout.widget.ConstraintLayout
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <TextView
+      android:id="@+id/profile_type_title"
+      style="@style/OnboardingProfileTypeHeaderStyle"
+      android:text="@string/onboarding_profile_type_activity_header"
+      app:layout_constraintBottom_toTopOf="@id/profile_type_learner_navigation_container"
+      app:layout_constraintEnd_toEndOf="parent"
+      app:layout_constraintStart_toStartOf="parent" />
+
+    <androidx.constraintlayout.widget.Guideline
+      android:id="@+id/profile_type_center_guide"
+      android:layout_width="wrap_content"
+      android:layout_height="wrap_content"
+      android:orientation="horizontal"
+      app:layout_constraintGuide_percent="0.30" />
```

Comment:
I wonder how we set those guideline percentages?
It seems there are various percentages on different screens, is it necessary to set a standard for them to keep a consistent look?

![Screenshot 2024-04-16 153147](https://github.com/oppia/oppia-android/assets/74568012/c3b18d32-4905-44f2-b15f-8d6951e26d01)

Reply:
At the moment, we don't have any defined standards regarding guidelines. So it is at the developer's discretion to use what will make the UI look closest to the mocks. We can however open a discusion with the team regarding this.